tools/src/main/java/org/apache/kafka/tools/TransactionalMessageCopier.java

```diff
@@ -402,7 +414,7 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
                     } catch (KafkaException e) {
                         log.debug("Aborting transaction after catching exception", e);
                         abortTransactionAndResetPosition(producer, consumer);
-                    }
+                    } 
```

Comment:
- https://github.com/junit-team/junit-framework/blob/575f741c2c73dd4242b29b8d4642fda9dfdd8cce/gradle/plugins/common/src/main/kotlin/junitbuild.spotless-conventions.gradle.kts#L39
- https://docs.openrewrite.org/recipes/java/format/removetrailingwhitespace
- https://github.com/apache/kafka/pull/21168

kindly related example to improve setup, streamline development convention.


```suggestion
                    }
```